README.md

```diff
@@ -16,6 +16,7 @@
 
 ## Coding for Beginners
 - [Funny Algorithms](https://github.com/ReciHub/FunnyAlgorithms)
+- [A First Timers Guide to an Open Source Project](https://auth0.com/blog/a-first-timers-guide-to-an-open-source-project/)
```

Comment:
Since it is a guide and now the title calls Repositories coding for beginners, I suggest creating a new section called "resources"